### PR TITLE
add shfmt ci check

### PR DESCRIPTION
**Pull request by asherf**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ jobs:
           python-version: ${{ env.PYTHON_VERSION }}
       - name: Shellcheck
         run: shellcheck pantsup.sh
+      - uses: luizm/action-sh-checker@v0.5.0
```
Comment (benjyw) [inline .github/workflows/ci.yml:29]: This runs on every `.sh` file?

**Review by benjyw — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (asherf) [reply .github/workflows/ci.yml:29]: yes. will add some PR to address the issues detected by this check.
see: https://github.com/pantsbuild/setup/pull/139 (first one), will have more.
so this PR can pass CI checks.

**Review by benjyw — COMMENTED** [review]
Looking at the results in the other PRs - the tab indents are crushing my soul. We could run with `-i 4` which will use 4-space indents, which I think all would greatly prefer. For one thing, everyone's editors are set up to emit spaces when you hit tab, so actually getting those tabs right would be impossible without running the tool.

**Comment by benjyw** [comment]
I have a pending change to use Pants in this repo, which will give us shfmt, shellcheck and all the rest for free. So that might be a better path than manually adding this sort of thing.

**Comment by benjyw** [comment]
Closing this due to https://github.com/pantsbuild/setup/pull/142

**Pull request closed without merging**